frontend/components/forms/fields/InputFieldWithIcon/InputFieldWithIcon.jsx

```diff
@@ -42,9 +41,9 @@ class InputFieldWithIcon extends InputField {
         data-has-tooltip={!!tooltip}
       >
         {tooltip ? (
-          <TooltipWrapper tipContent={tooltip}>{label}</TooltipWrapper>
+          <TooltipWrapper tipContent={tooltip}>{error || label}</TooltipWrapper>
```

Comment:
Is it correct that the UX is now that we continue to show the tooltip even when the label has changed to an error message? To me it would make more sense to not show the tooltip message anymore when the error is displayed.